### PR TITLE
Create yaml for Gov Patrick Murphy (VAN-50436)

### PR DESCRIPTION
**Pull request by OdeclasV**
Gov Patrick Murphy was recently sworn in, thus a new yaml had to be created for him. Most of this info/yaml code comes from former Gov Christie's yaml, but I checked all selectors and information to make to sure it matched the new form.

### CODE REVIEW

File: members/SNJG8473b29e.yaml [hunk 1]
```diff
@@ -0,0 +1,144 @@
+bioguide: XXXX
```
Comment (tomdionesotes) [inline members/SNJG8473b29e.yaml]: need to add in the correct bioguide

File: members/SNJG8473b29e.yaml [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+contact_form:
+  method: post
+  action: /cgi-bin/governor/govmail/govmail_1c_new.pl
```
Comment (tomdionesotes) [inline members/SNJG8473b29e.yaml]: this can just be `action: ""`

File: members/SNJG8473b29e.yaml [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+            "Public Utilities": "Public Utilities"
+            "Requesting Proclamations / Special Letter": "Requesting Proclamations / Special Letter"
+            "Superstorm Sandy/CDBG": "Superstorm Sandy/CDBG"
+            "Taxation": "Taxation"
+            "Temporary Disability": "Temporary Disability"
+            "Transportation / Motor Vehicles": "Transportation / Motor Vehicles"
+            "Unclaimed Property": "Unclaimed Property"
+            "Unemployment": "Unemployment"
+    - click_on:
+        - value: Continue...
```
Comment (tomdionesotes) [inline members/SNJG8473b29e.yaml]: I try to have `click_on` steps only have a `selector` and no `value` - makes the yaml a bit easier to read IMO

File: members/SNJG8473b29e.yaml [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+          selector: input[name='last']
+          value: $NAME_LAST
+          required: true
+        - name: street
+          selector: input[name='address1']
+          value: $ADDRESS_STREET
+          required: true
+        - name: street2
+          selector: input[name='address2']
+          value: $ADDRESS_STREET_2
```
Comment (tomdionesotes) [inline members/SNJG8473b29e.yaml]: could you add `required: false` here?

File: members/SNJG8473b29e.yaml (hunk 3 above)
Comment (OdeclasV) [reply members/SNJG8473b29e.yaml]: Okok. So essentially like

- click_on:
  selector: form#govmail ... 

right? 

This click_on + value pair was carried over from Christie's yaml

File: members/SNJG8473b29e.yaml (hunk 1 above)
Comment (OdeclasV) [reply members/SNJG8473b29e.yaml]: good call!

File: members/SNJG8473b29e.yaml (hunk 3 above)
Comment (OdeclasV) [reply members/SNJG8473b29e.yaml]: I edited to 
`click_on:
  selector: form#govmail button[name='Submit'] `

but the test failed. Not sure if that yaml code is correct. 

I put `value` back in there and test succeeded.

Comment (tomdionesotes) [reply members/SNJG8473b29e.yaml]: need the dash and correct indentation still

Comment (OdeclasV) [reply members/SNJG8473b29e.yaml]: yeap, my bad.  Added the dash and the correct indentation and the test passed. Will push the edited yaml now

File: members/SNJG8473b29e.yaml [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+            "Public Utilities": "Public Utilities"
+            "Requesting Proclamations / Special Letter": "Requesting Proclamations / Special Letter"
+            "Superstorm Sandy/CDBG": "Superstorm Sandy/CDBG"
+            "Taxation": "Taxation"
+            "Temporary Disability": "Temporary Disability"
+            "Transportation / Motor Vehicles": "Transportation / Motor Vehicles"
+            "Unclaimed Property": "Unclaimed Property"
+            "Unemployment": "Unemployment"
+    - click_on:
+      - selector: form#govmail button[name='Submit']
```
Comment (tomdionesotes) [inline members/SNJG8473b29e.yaml]: indentation here doesn't match the `fill_in` indentation below

File: members/SNJG8473b29e.yaml (hunk 4 above)
Comment (OdeclasV) [reply members/SNJG8473b29e.yaml]: This was added. Not sure why this comment is still open

Comment (tomdionesotes) [reply members/SNJG8473b29e.yaml]: cool, i see it below

**Pull request merged**